Gemfile

```diff
@@ -13,14 +13,23 @@ gem 'jbuilder', '~> 2.0'
 gem 'dotenv-rails'
 gem 'therubyracer'
 gem 'less-rails'
+gem 'rails-i18n'
+
+# views
 gem 'twitter-bootstrap-rails'
 gem 'font-awesome-less'
+gem 'slim'
+
```

Comment:
Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `font-awesome-less` should appear before `twitter-bootstrap-rails`.